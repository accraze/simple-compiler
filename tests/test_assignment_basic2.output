parsing new stmt
ttype = 25
parsing Assign Statement
parsing new stmt
ttype = 25
parsing Assign Statement
parsing new stmt
ttype = 6
WE HAVE A PRINT STM
print var a
have another??
ttype 6
token print
parsing new stmt
ttype = 6
WE HAVE A PRINT STM
print var b
have another??
ttype 25
token a
parsing new stmt
ttype = 25
parsing Assign Statement
parsing new stmt
ttype = 25
parsing Assign Statement
parsing new stmt
ttype = 6
WE HAVE A PRINT STM
print var a
have another??
ttype 6
token print
parsing new stmt
ttype = 6
WE HAVE A PRINT STM
print var b
have another??
ttype 21
token b
Returning Print Statement
Exiting Statement Node
Returning Assign Statement
Returning Assign Statement
Exiting Statement Node
Exiting Statement Node
Returning Assign Statement
Returning Assign Statement
457
221
65537
12481632
