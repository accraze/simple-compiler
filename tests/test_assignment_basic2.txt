
a , b  ;

{
a = 457 ;
b = 221 ; 
print a ;
print b ;
a = 65537 ; 
b = 12481632 ; 
print a ;
print b ;
}
