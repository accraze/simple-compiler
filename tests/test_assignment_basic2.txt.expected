457
221
65537
12481632

